code/web/src/modules/user/ProfileForm.js

```diff
@@ -36,8 +37,10 @@ class ProfileForm extends PureComponent {
     })
   }
 
-  onSubmit = () => {
+  onSubmit = (event) => {
+    event.preventDefault()
     updateUser(this.state)
+    this.props.toggleModal()
```

Comment:
Great job figuring out how to get this to work!

Reply:
^^!!